dwave/optimization/symbols.pyx

```diff
@@ -2329,12 +2189,12 @@ cdef class LP(Symbol):
         with zf.open(fname, mode="w", force_zip64=True) as f:
             np.save(f, states, allow_pickle=False)
 
-    cdef cppLPNode* ptr
+    cdef cppLinearProgramNode* ptr
 
-_register(LP, typeid(cppLPNode))
+_register(LinearProgram, typeid(cppLinearProgramNode))
 
 
-cdef class LPFeasible(ArraySymbol):
+cdef class LinearProgramFeasible(ArraySymbol):
     """Return whether the parent LP symbol's current solution is feasible.
```

Comment:
Might want to use `LinearProgram` in these docstrings?

Reply:
Chose the more succinct wording for now. "LP symbol" is descriptive, if we were doing
```
:class:`LinearProgram`
```
I would be more explicit